docs/import-haveno.md

```diff
@@ -1,4 +1,36 @@
-# Importing Bisq into IntelliJ IDEA
+## Importing Haveno into development environment
+
+This document describes how to import Haveno into an integrated development environment (IDE).
+
+## Importing Haveno into Eclipse IDE
+
+The following steps describe how to import Haveno into Eclipse IDE for development. You can also develop using [IntelliJ IDEA](#importing-haveno-into-intellij-idea) or VSCode if you prefer.
+
+> Note: Use default values unless specified otherwise.
+
+1. If you haven't already, clone Haveno projects:
+    1. `cd ~/git/`
```

Comment:
I would remove this step entirely. We can assume the dev will have their own environment set up where they will clone the two repositories.